docs/pom.xml

```diff
@@ -40,11 +40,96 @@
                         </goals>
                         <configuration>
                             <target>
-                                <exec dir="${project.basedir}" executable="jekyll" failonerror="true">
+                                <available file="${project.build.directory}/che-repo" property="cheRepoExists" type="dir"/>
```

Comment:
You cannot build in codenvy.io unless you port bind an external docker daemon to the workspace machine.